vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/main/java/com/vaadin/flow/data/performance/ObjectSizeCalculator.java

```diff
@@ -0,0 +1,432 @@
+// COPIED from https://github.com/twitter-archive/commons/blob/master/src/java/com/twitter/common/objectsize/ObjectSizeCalculator.java
+// to replace usage of class jdk.nashorn.internal.ir.debug.ObjectSizeCalculator not available in new JDKs
+// =================================================================================================
+// Copyright 2011 Twitter, Inc.
```

Comment:
This file looks like an exact copy of `jdk.nashorn.internal.ir.debug.ObjectSizeCalculator`, originally licensed under GPL2 by Oracle. Even if it's a derivative, GPL2 prohibits license change.